mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -164,6 +164,9 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 3 world
 		ARG 4 context
 		ARG 5 collisions
+	METHOD method_20802 getFireTime ()I
+	METHOD method_20803 setFireTime (I)V
+		ARG 1 value
```

Comment:
We tend to name thse `fireTime`/`fireTicks` instead of just plain `value`. Applies for all other mappings

Reply:
https://github.com/FabricMC/yarn/pull/880/commits/9b33165535fd4e3ed55e6b4793cb2c6aa36e943f

Reply:
https://github.com/FabricMC/yarn/commit/1aa7f82b723458a80bb0f7a6bbe2198bfffefa22